docs(secret-manager): faq (#1558)
scaleway/docs-content: faq/secret-manager.mdx

## In which regions is Secret Manager available?

The Secret Manager service is deployed in all Scaleway regions: PAR, WAW and AMS. Stored secrets are replicated in different zones within a region.

## How does Secret Manager guarantee the security of my secrets?

Secret Manager allows you to control access to secrets through IAM policies. When you retrieve a secret, Secret Manager decrypts it and transmits it securely via TLS to your local environment. At no point does the service write or cache secrets in plain text on permanent storage.

## How does Scaleway Secret Manager Scaleway encrypt my secrets?

Secret Manager Scaleway uses envelope encryption (AES-256 encryption algorithm) to encrypt your secrets with an internal Key Management Service (KMS).
To understand the secrets encryption process, refer to our [related documentation](/identity-and-access-management/secret-manager/reference-content/data-encryption-with-secret-manager/)

## How is Scaleway Secret Manager billed?

At the end of the month, you are billed for the number of secret versions stored and API requests made on the service.

> A secret version is billed if it is in an enable or disable state.

Reviewer: ```suggestion
A secret version is billed if it is in an enabled or disabled state.
```

> If you have used a secret version for five days, you will only be billed for the five days and not for the whole month.

Author: ```suggestion
If you have used a secret version for five days, you will only be billed for the five days and not for the whole month. Find out more about pricing on our [dedicated page](https://www.scaleway.com/en/pricing/?tags=available).
```

> If you have used a secret version for five days, you will only be billed for the five days and not for the whole month. Find out more about pricing on our [dedicated page](https://www.scaleway.com/en/pricing/?tags=available).

Reviewer: ```suggestion
If, for example, you have used a secret version for five days, you will only be billed for the five days and not for the whole month. Find out more about pricing on our [dedicated page](https://www.scaleway.com/en/pricing/?tags=available).
```

## In which regions is Secret Manager available?

The Secret Manager service is deployed in all Scaleway regions: PAR, WAW and AMS. Stored secrets are replicated in different zones within a region.

## How does Secret Manager guarantee the security of my secrets?

Secret Manager allows you to control access to secrets through IAM policies. When you retrieve a secret, Secret Manager decrypts it and transmits it securely via TLS to your local environment. At no point does the service write or cache secrets in plain text on permanent storage.

## How does Scaleway Secret Manager Scaleway encrypt my secrets?

Secret Manager Scaleway uses envelope encryption (AES-256 encryption algorithm) to encrypt your secrets with an internal Key Management Service (KMS).
To understand the secrets encryption process, refer to our [related documentation](https://www.scaleway.com/en/docs/identity-and-access-management/secret-manager/reference-content/data-encryption-with-secret-manager/)

## How is Scaleway Secret Manager billed?

At the end of the month, you are billed for the number of secret versions stored and API requests made on the service.

> ## How do I migrate my secrets to Secret Manager Scaleway?

Reviewer: ```suggestion
## How do I migrate my secrets to Scaleway's Secret Manager?
```

**To migrate your existing secrets to Secret Manager:**

1. Identify the secrets and their associated permissions

> 2. Create the API, CLI or Console secrets.

Author: I removed this part because it was not ready

To access the secrets stored in Secret Manager, you must create an Identity and Access Management (IAM) policy for your application. Then, you can replace the plain text secrets with code to retrieve the stored secrets through the Secret Manager API. For more information, see the [related documentation](https://www.scaleway.com/en/docs/identity-and-access-management/secret-manager/quickstart/).

## In which regions is Secret Manager available?

The Secret Manager service is deployed in all Scaleway regions: PAR, WAW and AMS. Stored secrets are replicated in different zones within a region.

## How does Secret Manager guarantee the security of my secrets?

Secret Manager allows you to control access to secrets through IAM policies. When you retrieve a secret, Secret Manager decrypts it and transmits it securely via TLS to your local environment. At no point does the service write or cache secrets in plain text on permanent storage.

## How does Scaleway Secret Manager Scaleway encrypt my secrets?

Secret Manager Scaleway uses envelope encryption (AES-256 encryption algorithm) to encrypt your secrets with an internal Key Management Service (KMS).

> To understand the secrets encryption process, refer to our [related documentation](https://www.scaleway.com/en/docs/identity-and-access-management/secret-manager/reference-content/data-encryption-with-secret-manager/)

Author: ```suggestion
To understand the secrets encryption process, refer to our [related documentation](/identity-and-access-management/secret-manager/reference-content/data-encryption-with-secret-manager/)
```

## How is Scaleway Secret Manager billed?

> 

Reviewer 2: ```suggestion
```

At the end of the month, you are billed for the number of secret versions stored and API requests made on the service.

> 

Reviewer 2: ```suggestion
```

> If you have used a secret version for five days, you will only be billed for the five days and not for the whole month.

Author: ```suggestion
If you have used a secret version for five days, you will only be billed for the five days and not for the whole month. Find out [pricing information](https://www.scaleway.com/en/pricing/) about Secret Manager.
```
